framework/CodeInterfaces/Dymola/DymolaInterface.py

```diff
@@ -315,22 +331,23 @@ def finalizeCodeOutput(self, command, output, workingDir):
       @ In, workingDir, string, current working dir
       @ Out, output, string, optional, present in case the root of the output file gets changed in this method.
     """
-    self._vars = {}
-    self._blocks = []
-    self._namesData1 = []
-    self._namesData2 = []
-    self._timeSeriesData1 = []
-    self._timeSeriesData2 = []
+    _vars = {}
```

Comment:
why this was in self?
This should remove the memory build up that we have seen in the past.
=> Good catch.

Reply:
No idea... O_o